src/components/Experiment/QuickOutput/Outputs/TextGuidedImagetoImage/TextGuidedImageToImage.stories.js

```diff
@@ -1,15 +1,14 @@
-import React from 'react';
-import { TestTextGuidedImageToImage } from './testData/testTextGuidedImagetoImageOutput'
-import { TextGuidedImageToImageOutput } from './TextGuidedImagetoImageOutput'
+import TextGuidedImageToImageOutput from "./TextGuidedImageToImageOutput";
```

Comment:
@ShivanshShalabh Somehow this file got deleted during the merge, so I just re-added it.